products/samsungmobile.md

```diff
@@ -30,27 +135,112 @@ releases:
     support: true
     eol: false
     releaseDate: 2022-04-01
+    
+-   releaseCycle: "Galaxy F23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-E236B/INS/doc.html
+    releaseDate: 2022-03-16
+
+-   releaseCycle: "Galaxy M33"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M336BU/INS/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy M23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M236B/XXV/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy A23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-A235M/PET/doc.html
+    releaseDate: 2022-03-25
+
+-   releaseCycle: "Galaxy A13"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-A135M/PET/doc.html
+    releaseDate: 2022-03-23
+
+-   releaseCycle: "Galaxy S22 Ultra 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S908E/VAU/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22+ 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S906E/XXV/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S901E/XXV/doc.html
+    releaseDate: 2022-02-25
 
+-   releaseCycle: "Galaxy Tab S8 Ultra"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X906B/XXV/doc.html
+    releaseDate: 2022-04-30
+
+-   releaseCycle: "Galaxy Tab S8+"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X806B/XXV/doc.html
+    releaseDate: 2022-04-14
+
+-   releaseCycle: "Galaxy Tab S8"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
Not sure about this one, according to https://www.gsmarena.com/samsung_galaxy_tab_s8-11343.php this model is upgradable to Android 13.

Reply:
Confirmed on https://www.sammobile.com/news/every-samsung-galaxy-device-updated-to-android-13-one-ui-5-0/.

This is not the only one that should not have been marked out of support, this is also the case for Galaxy F42 5G, Galaxy M52 5G, Galaxy M22...

Reply:
I wrote a quick script (committed in this PR, can be removed later) that check if devices has a support date and are listed in https://www.sammobile.com/news/every-samsung-galaxy-device-updated-to-android-13-one-ui-5-0/. I think those devices should be marked as supported :

```
$ python3 _auto/samsung.py 
Galaxy F42 5G (2022-10-01)
Galaxy M52 5G (2022-10-01)
Galaxy M22 (2022-10-01)
Galaxy M32 5G (2022-10-01)
Galaxy A03s (2022-10-01)
Galaxy F22 (2022-10-01)
Galaxy M32 (2022-10-01)
Galaxy A22 5G (2022-10-01)
Galaxy A22 (2022-10-01)
Galaxy Tab S7 FE (2022-10-01)
Galaxy M42 5G (2022-10-01)
Galaxy M62 (2022-10-01)
Galaxy F62 (2022-10-01)
Galaxy A32 5G (2022-10-01)
Galaxy M12 (2022-10-01)
Galaxy Tab S7 (2022-10-01)
Galaxy Tab S6 Lite (2022-10-01)
Galaxy S10 Lite (2022-10-01)
```

Reply:
I agree with you, I believe this was a misconception in my part, we cannot assume a device is no longer supported if it just received the latest android version, we can only assume a device is no longer supported if it doesn't receive the next android version.

Reply:
Also checked products marked as unsupported less than one year after their release date, got :

```
$ python3 _auto/samsung.py 
Galaxy A13 (192)
Galaxy Tab S8 Ultra (154)
Galaxy Tab S8+ (170)
Galaxy Tab S8 (193)
Galaxy Tab A8 10.5 (2021) (257)
Galaxy A03 (253)
Galaxy A03 Core (299)
```

All those are still supported according to https://www.sammobile.com/news/every-samsung-galaxy-device-updated-to-android-13-one-ui-5-0/ or https://www.gsmarena.com/.

The script is :

```python
import frontmatter
from datetime import date

with open("products/samsungmobile.md", 'r') as f:
    product = frontmatter.load(f)
    for cycle in product['releases']:
        name = cycle['releaseCycle']
        support = cycle['support']
        release = cycle['releaseDate']
        if isinstance(support, date) and ((support - release).days < 365):
            print(f"{name} ({(support - release).days})")
```

Reply:
Models updated, resolving this conversation.